sqlitecluster/SQLite.cpp

```diff
@@ -183,8 +183,10 @@ void SQLite::commonConstructorInitialization(bool hctree) {
     sqlite3_trace_v2(_db, SQLITE_TRACE_STMT, _sqliteTraceCallback, this);
 
     // Update the cache. -size means KB; +size means pages
-    SINFO("Setting cache_size to " << _cacheSize << "KB");
-    SQuery(_db, "increasing cache size", "PRAGMA cache_size = -" + SQ(_cacheSize) + ";");
+    if (_cacheSize) {
+        SINFO("Setting cache_size to " << _cacheSize << "KB");
+        SQuery(_db, "increasing cache size", "PRAGMA cache_size = -" + SQ(_cacheSize) + ";");
```

Comment:
NAB do we want a "Not increasing cache_size" log here since there are no other logs in this block that it will hit now?

Reply:
We will still log if queries are slow, I don't think this is helpful, it'll just be a logline every time we create a DB handle about something we're not doing. We already have logs from SQLitePool about creating DB handles.